doc/node.1

```diff
@@ -182,28 +182,36 @@ Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
 \',\'\-separated list of core modules that should print debug information.
 
 .TP
-.BR NODE_PATH =\fIpath\fR[:\fI...\fR]
-\':\'\-separated list of directories prefixed to the module search path.
+.BR NODE_DISABLE_COLORS =\fI1\fR
+When set to \fI1\fR, colors will not be used in the REPL.
 
 .TP
-.BR NODE_DISABLE_COLORS=1
-When set to 1 colors will not be used in the REPL.
+.BR NODE_EXTRA_CA_CERTS =\fIfile\fR
```

Comment:
It's in `v7.x-staging`  at least:

https://github.com/nodejs/node/blob/e07b6516d76ceec81e8606dff59698f7949fe63b/src/node.cc#L4539